objcgen/embedder.cs

```diff
@@ -279,6 +299,9 @@ public int Compile ()
 
 			BuildInfo[] build_infos;
 
+            var iosBitcodeFlags = BitcodeOption == BitcodeOption.Default ? "" : ( BitcodeOption == BitcodeOption.True ? "-fembed-bitcode" : "" );
+            var otherBitcodeFlags = BitcodeOption == BitcodeOption.Default ? "-fembed-bitcode" : ( BitcodeOption == BitcodeOption.True ? "-fembed-bitcode" : "" );
```

Comment:
spaces were used instead of tabs

Reply:
like mentioned in https://github.com/mono/Embeddinator-4000/pull/628#discussion_r176771994 this should be moved into the switch below. There's no need to always evaluate both options and the platform customization for `BuildInfo` is better kept in a single place.